test/release/examples/primers/arrayVectorOps.good

```diff
@@ -1,3 +1,17 @@
+arrayVectorOps.chpl:29: warning: push_front is deprecated - please use List.insert
```

Comment:
I think we should consider removing these examples from the primer, or at least adding a `.. warning ::` / `.. note::` that vector operations are being deprecated on arrays and users should refer to the Lists module for support of vector operations.

Reply:
I've made an issue to keep track of this. The primers will be updated when deprecation warnings are switched on by default.